### ISSUE TITLE
Feature Request/Help: Add 1-5

### ISSUE THREAD

**Issue by dice4321** [Ryuuhou/AHKanColle#23]
Now that OASW strikes has been added into the game, it would be nice to be able to go to 1-5 without using any buckets.

I made some progress on my fork project but for the pixel locations I just guessed for the extra button and it worked wooo,
now I ran into another problem, which is selecting the 'line abreast' option.

http://i.imgur.com/uTmPw7L.png 

I need some help determining what the x and y cordinates are @Ryuuhou!

**Comment by Ryuuhou** [comment]
try FX+266 and FY-115

**Comment by dice4321** [comment]
It works, thanks Ryuuhou! Should I push or nah?

**Comment by Ryuuhou** [comment]
Go for it, I'll most likely edit it to my liking before pushing to master (pull request on the dev branch please)

**Comment by dice4321** [comment]
Im still new to this github thing, I made the modifications using the master branch, if I do a pull request on the dev branch would it add the new lines automatically, or would I need to clone the dev branch, make changes there, then do a pull request?

**Comment by Ryuuhou** [comment]
Wait sorry, let me fix the dev branch

**Comment by Ryuuhou** [comment]
Try a pull request on the dev2 branch, I'll let you know if it works.  You shouldn't need to clone dev2, its the same as master right now.

**Comment by Ryuuhou** [comment]
I got it, thanks.  It is still possible to taiha in 1-5 though (4 subs on second to last node at lvl 99) so I would be careful about going all the way to the boss, unless I'm missing something.

**Comment by dice4321** [comment]
Yeah you're right, its more recommended for endless 1-5A run, since the ship with OASW's moral will never decrease, its nearly guaranteed to kill the single enemy in the first node. Don't know if you're more likely to be cat bombed, will need to do some testing. Because I ran mrmin123's for 8 hours, and I got catbombed, and his is slow as hell for a single node. But yours is blazing fast, and I ran it for 8 hours, and I didnt get catbombed. Could be because I used KCV for yous and KC3 for mrmin123?
